db/schema.rb

```diff
@@ -771,7 +771,8 @@
   end
 
   create_table "versions", force: :cascade do |t|
-    t.string "item_type", null: false
+    t.string "item_type"
+    t.string "{:null=>false}"
```

Comment:
Also a mistake?

Reply:
Yes, I had some trouble removing the database.yml, and I must had added this in error!sorry about that!